bundles/plugins/org.bonitasoft.studio.designer/src/org/bonitasoft/studio/designer/ui/property/section/control/FormReferenceExpressionValidator.java

```diff
@@ -51,8 +53,10 @@ public FormReferenceExpressionValidator(final RepositoryAccessor repositoryAcces
     public IStatus validate(final Object value) {
         final String content = inputExpression.getContent();
         final WebPageRepositoryStore repositoryStore = repositoryAccessor.getRepositoryStore(WebPageRepositoryStore.class);
-        return repositoryStore.getChild(content) == null ? ValidationStatus.error(Messages.bind(Messages.pageDoesntExists,
-                String.format("%s (%s)", inputExpression.getName(), content))) : ValidationStatus.ok();
+        final String errorMessage = ProcessPackage.Literals.RECAP_FLOW__OVERVIEW_FORM_MAPPING.equals(feature) ? Messages.bind(Messages.pageDoesntExists,
+                String.format("%s (%s)", inputExpression.getName(), content)) : Messages.bind(Messages.formDoesntExists,
```

Comment:
I think that using if then else would be easier to read as it is more complicated